nltk/sentiment/vader.py

```diff
@@ -88,18 +87,16 @@ def negated(input_words, include_nt=True):
     """
     Determine if input contains negation words
     """
-    neg_words = []
-    neg_words.extend(NEGATE)
-    for word in neg_words:
-        if word in input_words:
+    neg_words = NEGATE
+    for word in input_words:
+        if word.lower() in neg_words:
             return True
     if include_nt:
         for word in input_words:
-            if "n't" in word:
+            if "n't" in word.lower():
                 return True
-    if "least" in input_words:
-        i = input_words.index("least")
-        if i > 0 and input_words[i-1] != "at":
+    for i, word in enumerate(input_words):
+        if i > 0 and word.lower() == "least" and input_words[i-1].lower() != 'at':
```

Comment:
Honestly I thinkg something like pairwise from https://docs.python.org/3/library/itertools.html#itertools-recipes should be used here, but I'm not sure what code convetion is used in the library, so I made it similar to what was before
`
def pairwise(iterable):
    "s -> (s0,s1), (s1,s2), (s2, s3), ..."
    a, b = tee(iterable)
    next(b, None)
    return zip(a, b)
`